tests/generic_views/test_base.py

```diff
@@ -519,3 +520,11 @@ def test_template_mixin_without_template(self):
         view = views.TemplateResponseWithoutTemplate()
         with self.assertRaises(ImproperlyConfigured):
             view.get_template_names()
+
+
+class DirectlyAccessedLoginPageTest(unittest.TestCase):
```

Comment:
This test module is for `django.views.generic`. Tests for the login view go in `tests/auth_tests/test_views.py`.